build_test/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## 0.2.0+1
+## 0.3.0
```

Comment:
Ah good catch. At one point I had made a breaking change but ended up reverting it.